chainer/training/extensions/nan_killer.py

```diff
@@ -0,0 +1,21 @@
+from chainer.training import extension
+
+
+class NaNKiller(extension.Extension):
```

Comment:
I thought it is better to name it `NaNDetector` or sth, because it kills the training loop, not `NaN` itself (`NaNKiller` sounds like removing NaNs).